ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -251,6 +251,13 @@
       "The maximum number of concurrent requests allowed for pull "
       + "queries on this host. Once the limit is hit, queries will fail immediately";
 
+  public static final String KSQL_QUERY_PULL_MAX_HOURLY_BANDWIDTH_CONFIG
+      = "ksql.query.pull.max.hourly.bandwidth";
+  public static final Integer KSQL_QUERY_PULL_MAX_HOURLY_BANDWIDTH_DEFAULT = Integer.MAX_VALUE;
+  public static final String KSQL_QUERY_PULL_MAX_HOURLY_BANDWIDTH_DOC
+      = "The maximum amount of pull query bandwidth allowed over a period of one hour."
```

Comment:
You should specify that this is in MB in the comments.  For other disk limits, btw, it seems that we just do things in bytes.  Might want to do that for consistency.  If you do make that change, you probably want to change the config type to a long.  Also, it might make sense to tack on to the config name the unit, e.g. "ksql.query.pull.max.hourly.bandwidth.bytes" or "ksql.query.pull.max.hourly.bytes"